tests/single-instance.tftest.hcl

```diff
@@ -0,0 +1,134 @@
+provider "spacelift" {
+  api_key_endpoint = "https://masterpointio.app.spacelift.io"
```

Comment:
I'd move this to [SPACELIFT_API_KEY_ENDPOINT](https://registry.terraform.io/providers/spacelift-io/spacelift/latest/docs#SPACELIFT_API_KEY_ENDPOINT-1) secret env var.

Reply:
Yeah, you're right. Thanks for the callout.

Reply:
Will do this as a follow up shortly ⌛

Reply:
Fixed.